pages/agenda.tsx

```diff
@@ -46,15 +46,15 @@ class AgendaPage extends React.Component<AgendaPageProps> {
     const dates = this.props.pageMetadata.dates
 
     const AfterSessionDetails = (s: Session) => {
-      return (
+      return dates.AcceptingFeedback ? (
```

Comment:
Not a fan of?

```jsx
if (!dates.AcceptingFeedback) return null

return (
    <p className="text-center">
        <a className="btn btn-secondary" target="_blank" href={conference.SessionFeedbackLink}>
            Give feedback
        </a>
    </p>
)
```

Note, I also removed the unnecessary fragment as the `p` was a single-node return anyway.

Reply:
Sorry, missed this. Feel free to push that change onto my PR branch

Reply:
I've merged anyway, feel free to go back and correct :)

Reply:
Ha, forgot about this as well :) Will do!